### ISSUE TITLE
Add a hint that `lang` and `dir` attributes have been set

### ISSUE THREAD

**Issue by druxstr** [WordPress/gutenberg#51875]
## What problem does this address?

PR #49985 added `lang` and `dir` attributes to text-formatting tools (see #15349 for more context).

But when you do add on of these attributes, there's no hint (visual or else) that you did.

From the visual editor, you cannot know that the attributes have been correctly set (or set at all).

You have to switch to code editor to confirm that you added the attribute correctly, to the exact chain you targeted.

## What is your proposed solution?

Content that has a `lang` or `dir` attributes set should get some kind of visual hint (highlighting) for backend users.

This hint can't be _only_ visual though, as it would cause accessibility issues.
